### PR TITLE
[capi] fix periodics

### PR DESCRIPTION
**Pull request by chuckha**
Signed-off-by: Chuck Ha <chuckh@vmware.com>

/assign @detiber

### CODE REVIEW

File: config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml [hunk 1]
```diff
@@ -47,6 +47,11 @@ periodics:
   interval: 1h
```
Comment (detiber) [inline config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml]: Not directly related to this PR, but I'd really like to s/ci-/periodic-/ in the names of the periodic jobs.

File: config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml [hunk 2]
```diff
@@ -47,6 +47,11 @@ periodics:
...
   decorate: true
   path_alias: sigs.k8s.io/cluster-api
```
Comment (detiber) [inline config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml]: Do we still need the path_alias defined here?

File: config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml [hunk 3]
```diff
@@ -72,6 +77,11 @@ periodics:
     preset-dind-enabled: "true"
     preset-kind-volume-mounts: "true"
   path_alias: sigs.k8s.io/cluster-api
```
Comment (detiber) [inline config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml]: Do we still need the path_alias defined here?

File: config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml (hunk 1 above)
Comment (chuckha) [reply config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml]: might as well throw it in with this PR

**Pull request merged**